privacyidea/cli/pitokenjanitor/utils/findcontainer.py

```diff
@@ -0,0 +1,193 @@
+from typing import Generator
```

Comment:
I would call the file just `container.py`. The `find` command is just one of the commands.